Resources/Prototypes/Entities/Mobs/Player/human.yml

```diff
@@ -24,3 +24,20 @@
   - type: AiFactionTag
     factions:
     - NanoTrasen
+
+- type: entity
+  name: centcom official
+  parent: HumanMob_Content
+  id: HumanMob_CentcomOfficial
+  components:
+  - type: Icon
+    sprite: Interface/Misc/markers.rsi
+    state: Centcom
+  - type: GhostRoleMobSpawner
+    prototype: HumanMob_CentcomOfficial
+    deleteOnSpawn: true
+    makeSentient: false
+    name: centcom official
+    description: Inspect the station, jot down performance reviews for heads of staff, bug the Captain.
+  - type: UtilityAI
+    startingGear: CentcomGear
```

Comment:
Is this being used for the starting gear or should it wander.

Reply:
starting gear